docs/_tutorials/java-client.md

```diff
@@ -48,11 +51,11 @@ MsgVpnApi sempApiInstance = new MsgVpnApi(thisClient);
 
 Remember to update the values in the above example to match your environment.
 
-[*Source Reference: BasicOperationsSample.initialize()*]({{ site.repository }}/blob/master/src/main/java/com/solace/samples/BasicOperationsSample.java#L51){:target="_blank"}
+[*Source Reference: BasicOperationsSample.initialize()*]({{ site.repository }}/blob/45ff0a4e8974fd68b7cb72152c749327defcbfd1/java/src/main/java/com/solace/samples/BasicOperationsSample.java#L51-L60){:target="_blank"}
```

Comment:
We don't want to point to the branch. It may not be worth it to maintain these code links directly to line numbers. Let's discuss.